entrypoints/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+if [ ! $BOOTSTRAP == "" ];
+then
+  echo "Fetching configuration for $BOOTSTRAP network"
+  sleep 5
```

Comment:
Why a sleep here?

Reply:
Was there just for debugging purposes. Will remove all the sleeps